api/src/pcapi/core/finance/models.py

```diff
@@ -719,3 +725,100 @@ class PaymentMessage(Base, Model):
     id: int = sqla.Column(sqla.BigInteger, primary_key=True, autoincrement=True)
     name: str = sqla.Column(sqla.String(50), unique=True, nullable=False)
     checksum: bytes = sqla.Column(sqla.LargeBinary(32), unique=True, nullable=False)
+
+
+##
+# When an incident is `CREATED` it can be either `CANCELLED` and it's the end of this incident. Or it can be `VALIDATED`.
+# Once `VALIDATED`, the status becomes `UNDER COMPENSATION` to trigger the functions for payments.
+# `TERMINATED` means the incident is finished and all the payments completed
+##
+class IncidentStatus(enum.Enum):
+    CREATED = "CREATED"
+    VALIDATED = "VALIDATED"
+    UNDER_COMPENSATION = "UNDER COMPENSATION"
+    TERMINATED = "TERMINATED"
+    CANCELLED = "CANCELLED"
+
+
+class IncidentType(enum.Enum):
+    OVERPAYMENT = "OVERPAYMENT"
+    COMMERCIAL_GESTURE = "COMMERCIAL GESTURE"
+    OFFER_PRICE_REGULATION = "OFFER PRICE REGULATION"
+    FRAUD = "FRAUD"
+
+
+class Incident(Base, Model):
+    id: int = sqla.Column(sqla.BigInteger, primary_key=True, autoincrement=True)
+    type: IncidentType = sqla.Column(
```

Comment:
(nommage / question de goût) : je préfère utiliser le terme `kind`, car `type` est un built-in Python.